v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -209,5 +213,47 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.0/pca/gnomad.v4.0.pca_loadings.ht',
         },
     },
+    ReferenceDataset.gnomad_mito: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.MITO]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/3.1/ht/genomes/gnomad.genomes.v3.1.sites.chrM.ht',
+        },
+    },
+    ReferenceDataset.gnomad_non_coding_constraint: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/gnomad_nc_constraint/gnomad_non-coding_constraint_z_scores.ht',
```

Comment:
Wherever possible we should not be hosting reference datasets. If this is not publicly available we should add an explicit comment to the config explaining where we got the data that we put in the public bucket and why we are hosting the data ourselves

Reply:
So looking at the README in that file,  `constraint_z_genome_1kb.raw.download.txt.gz` seems to be everything, `constraint_z_genome_1kb.qc.download.txt.gz` seems to be everything that passed QC and constraint_z_enh_gene_roadmaplinks.all.download.txt.gz is z scores of enhancers for specific genes. The enhancer one is definitely wrong, unfortunately we did not document what we used when we copied that file so aI am also not sure which is better.
 According to the README theres ~2,500,000 raw scores and around ~2,000,000 passing QC scores. Maybe check the file we currently have in GCP and see if it has a number similar to either of those and if ti does use that and if not just use the raw scores

Reply:
we have 2,042,629 rows in our hosted reference dataset, so I'll use the raw scores

Reply:
true, I thought having fewer scores would be worse but I think it's a closer match. I used the passing scores file.